spring-boot-tools/spring-boot-loader/src/main/java/org/springframework/boot/loader/jar/CentralDirectoryFileHeader.java

```diff
@@ -118,7 +118,7 @@ public long getTime() {
 
 	/**
 	 * Decode MS-DOS Date Time details. See
-	 * <a href="http://mindprod.com/jgloss/zip.html">mindprod.com/jgloss/zip.html</a> for
+	 * <a href="https://mindprod.com/jgloss/zip.html">mindprod.com/jgloss/zip.html</a> for
```

Comment:
This page is actually only reachable via `http` but spring-javaformat checkstyle requires every Link to be `https`. Is there something to do about this? Maybe adding this info in the comment, too?